DevoxxRemoteFunctions/build.gradle

```diff
@@ -79,5 +126,13 @@ task buildAwsLambda {
     )
 }
 
+task uploadAwsLambda {
+    dependsOn(
```

Comment:
You could let this task depend on all the tasks of type AWSLambdaUpdateFunctionCodeTask:

    dependsOn tasks.withType(AWSLambdaUpdateFunctionCodeTask)